fix: do not remove voxtral from supported models (#6757)
scaleway/docs-content: pages/generative-apis/reference-content/supported-models.mdx

allenai/molmo-72b-0924:fp8
```

## Multimodal models (Text and Audio)

### Voxtral-small-24b-2507
Voxtral-small-24b-2507 is a model developed by Mistral to perform text processing and audio analysis on many languages.

> This model was optimized to enable transcription in many languages while keeping conversational capabilities (translations, classification, etc

Reviewer: ```suggestion
This model was optimized to enable transcription in several languages while keeping conversational capabilities (translations, classification, etc
```

)

| Attribute | Value |
|-----------|-------|
| Provider | Mistral |
| Supports structured output | Yes |
| Supports function calling |  Yes |
| Supports parallel tool-calling | Yes |
| Supported audio formats | `flac`, `m4a`, `mpeg`, `mp2`, `mp3`, `mp4`, `ogg`, `wav`, `webm` |
| Audio chunk duration | 30 seconds |
| Token duration (audio)| 80ms |
| Maximum transcription duration| 30 minutes |
| Maximum understanding duration| 40 minutes |
| Maximum file size - Serverless | 25 MB |
| Supported languages | English, French, German, Dutch, Spanish, Italian, Portuguese, Hindi |
| Compatible Instances (max context in tokens\*) - Dedicated Deployment | H100, H100-2 |
| Hugging Face model card | [voxtral-small-24b-2507](https://huggingface.co/mistralai/Voxtral-Small-24B-2507) |

> \*Maximum context length is only mentioned when an instance's VRAM size limits context length. Otherwise, maximum context length is the one def

Reviewer: ```suggestion
\*Maximum context length is only mentioned when an instance's VRAM size limits context length. Otherwise, maximum context length is the one defined by the model.
```

> ned by the model.

Reviewer: ```suggestion

```
